docs/source/host-collect-analyze/journald.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Journald
+description: Collect logs from journald
+---
+
+## Journald Collector
+
+To collect log entries from the journald service, you can use the `journald` collector.
+
+### Parameters
+
+In addition to the [shared collector properties](/collect/collectors/#shared-properties), the `journald` collector accepts the following parameters to filter journal records:
+
+#### `System` (Optional)
```

Comment:
the headings here should match the case of the parameter:

#### `system` (Optional)

#### `dmesg` (Optional)

Reply:
![Screenshot 2024-07-29 at 12 18 17](https://github.com/user-attachments/assets/195a3678-2a62-4d16-8e7e-fbbc96ce7ba4)

Reply:
many thanks @adamancini! I've updated the headings